onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java

```diff
@@ -150,7 +153,7 @@ private long getReminderMS(String tripId, String stopId) {
     // Return the difference between now and the predicted/scheduled
```

Comment:
Since we're modifying this method, can you please fix this Javadoc?  If I'm reading the code right, this doesn't return the difference between now and predicted/scheduled time - instead it returns either the predicted or scheduled time of this particular `tripId`, or null if that tripId can't be found.